internal/experimental/registry/constants.go

```diff
@@ -31,4 +31,7 @@ const (
 
 	// ProvLayerMediaType is the reserved media type for Helm chart provenance files
 	ProvLayerMediaType = "application/vnd.cncf.helm.chart.provenance.v1.prov"
+
+	// LegacyChartContentLayerMediaType is the legacy reserved media type for Helm chart package content.
+	LegacyChartContentLayerMediaType = "application/tar+gzip"
```

Comment:
Can we name this `LegacyChartLayerMediaType` to match `ChartLayerMediaType`?

Reply:
ack